src/v3/src/transformer/terminal/transformUnhandledErrors.ts

```diff
@@ -42,7 +42,7 @@ const getWidgetMessage = (
 
         // TODO: re-visit, handle side effects in hooks
         // If the session expired, this clears session to allow new transaction bootstrap
-        if (widgetProps && message.i18n.key === 'idx.session.expired') {
+        if (widgetProps && message?.i18n?.key === 'idx.session.expired') {
```

Comment:
Not related to current task. But while testing various scenarios, I has an error from backend without `i18n` key:
```
{
  "messages": {
    "type": "array",
    "value": [
      {
        "message": "Conflicting User IDs specified on state token",
        "class": "ERROR"
      }
    ]
  },
  ..
}
```
Without optional chaining, there will be runtime error